instagram_challenge/spec/features/likes_features_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+feature 'liking comments' do
+  before do
+    nyc = Photo.create(caption: 'NYC')
+  end
+
+  # scenario 'a user can endorse a review, which updates the review endorsement count', js: true do
+  #   visit '/photos'
+  #   click_link 'Like Comment' #are we endorsing restaurants or the review of the restaurants?
```

Comment:
Metrics/LineLength: Line is too long. [95/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)